Fix EZP-25580: eZSupportTools PHP collector service naming fix

glye opened the pull request:
> Fixes https://jira.ez.no/browse/EZP-25580
> Status: Ready for review

The service is named `support_tools.info_collectors.php`, but to be consistent with the class name and the hardware collector it should be named `support_tools.info_collectors.ezc_php`, as it uses the Components.

- [x] squash

glye commented:
Tiny review ping @bdunogier

bdunogier commented:
Before diving into larger considerations below, I'd reverse the naming here: `support_tools.info_collectors.php.ezc` (`ezc` implementation of the `php` `info_collector`).

That aside, +1.

bdunogier commented on Resources/config/services.yml:
```diff
@@ -8,8 +8,8 @@ services:
 
-    support_tools.info_collectors.php:
-        class: %support_tools.info_collectors.php.class%
+    support_tools.info_collectors.ezc_php:
```
`support_tools.info_collectors.php.ezc` (from least to most specialized).

glye commented:
Renamed:
doctrine_database -> database.doctrine
ezc_hardware -> hardware.ezc
ezc_php -> php.ezc
support_tools.info_collector.system_info.ezc_system_info -> support_tools.info_collector.system_info.ezc
Rebased on master

glye commented:
@bdunogier larger considerations?

bdunogier commented:
> @bdunogier larger considerations?

Hum, this one got lost somewhere :D

bdunogier commented:
No larger considerations, +1, feel free to merge :-)
